packages/ui/Routable/Link.js

```diff
@@ -7,11 +7,70 @@ import {Component} from 'react';
 
 import useLink from './useLink';
 
+/**
+ * A component that is used to make a trigger to navigate inside of {@link ui/Routable.Routable} component.
+ *
+ * In the following example, `Sample` would render `Main` with two Links for `About` and `FAQ`.
+ * If `About` is clicked, the content of `About` would be displayed under `Main`.
+ *
+ * ```
+ * const Views = Routable({navigate: 'onNavigate'}, ({children}) => <div>{children}</div>);
+ *
+ * const Main = () => (
+ *	  <div>
+ *		  <Link path="./about">About</Link>
+ *		  <Link path="./faq">FAQ</Link>
+ *	  </div>
+ * );
+ *
+ * const About = () => (<div>Greetings! We are Enact team.</div>);
+ *
+ * const Faq = () => (<div>List of FAQ</div>);
+ *
+ * const Sample = (props) => {
+ *	  // use 'main' for the default path
+ *	  const [path, nav] = useState('main');
+ *	  // if onNavigate is called with a new path, update the state
+ *	  const handleNavigate = useCallback((ev) => nav(ev.path), [nav]);
+ *
+ *	  return (
+ *		  <Views {...props} path={path} onNavigate={handleNavigate}>
+ *			  <Route path="main" component={Main}>
+ *				  <Route path="about" component={About} />
+ *				  <Route path="faq" component={Faq} />
+ *			  </Route>
+ *		  </Views>
+ *	  );
+ * };
+ * ```
+ *
+ * @class Link
+ * @ui
+ * @memberof ui/Routable
+ * @public
+ */
 const LinkBase = kind({
```

Comment:
I guess you would add some basic private documents for `LinkBase`.

Reply:
Right, I've added a description for LinkBase.